addon/components/bourbon-search-select-field.js

```diff
@@ -34,7 +34,6 @@ export default Component.extend(SelectMixin, {
   autofocus: null,
   readonly: null,
   disabled: false,
-  hasValue: computed.notEmpty('value'),
```

Comment:
I assume this and a few other lines are just removing `hasValue` because it's unused?

Reply:
yep.  :)